packages/plugins/ansiblerole-satellite-receptor-installer/ansiblerole-satellite-receptor-installer.spec

```diff
@@ -39,6 +39,9 @@ cp -pR %{repo_name}-%{version} %{buildroot}%{_datadir}/ansible/roles/%{role_orgn
 %doc %{repo_name}-%{version}/README.md
 
 %changelog
+* Tue Nov 12 2020 Marek Hulan <mhulan@redhat.com> - 0.6.0-2
```

Comment:
It's Mar not Nov :)

Reply:
also not Tuesday :-) I updated previous log too, hopefully that's ok

Reply:
also who says I haven't sent this from future?